packages/api/src/record.ts

```diff
@@ -295,6 +350,91 @@ export class Record implements RecordModel {
     return dataObj;
   }
 
+  private async _processMessage(options: any){
+
+    const { store = true } = options;
+
+    const request = {
+      messageType : DwnInterfaceName.Records + DwnMethodName.Write,
+      author      : this._connectedDid,
+      target      : this._connectedDid,
+      import      : !!options.import,
+      store
+    } as any;
+
+    if (options.rawMessage) {
+      removeUndefinedProperties(options.rawMessage);
+      request.rawMessage = options.rawMessage as Partial<RecordsWriteMessage>;
+    }
+    else {
+      request.messageOptions = options.messageOptions;
+    }
+
+    if (options.dataStream) {
+      request.dataStream = options.dataStream;
+    }
+
+    return this._agent.processDwnRequest(request);
+  }
+
+
+  async store(options: any = {}): Promise<any> {
```

Comment:
Please add some documentation that explains what this does.

```suggestion
  async store(options: any = {}): Promise<any> {
```

Reply:
Added